stories/documentation/forms/fields/checkbox/angular/checkboxfield.stories.ts

```diff
@@ -30,7 +30,7 @@ export default {
 
 export const Basic: StoryObj<CheckboxInputComponent & FormFieldComponent> = {
 	render: (args, { argTypes }) => {
-		const { label, hiddenLabel, tooltip, inlineMessage, inlineMessageState, size, ...inputArgs } = args;
+		const { label, hiddenLabel, tooltip, inlineMessage, inlineMessageState, size, mixed, ...inputArgs } = args;
```

Comment:
Keeping mixed in `inputArgs` makes more sense so it's handled as the boolean attribute it is would be better.